DockerfileTomcat

```diff
@@ -1,4 +1,7 @@
 ARG VERSION
 FROM tomcat:$VERSION
-RUN  rm -fr /usr/local/tomcat/webapps/ROOT
+RUN  apt-get update && \
+     apt-get install -y curl && \
+     apt-get clean && \
+rm -fr /usr/local/tomcat/webapps/ROOT
```

Comment:
Leave `RUN` as is, this will create another layer, but it will be very light and will document this change.